src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1698,6 +1731,9 @@ def __call__(
         if head_mask is None:
             head_mask = jnp.ones((self.config.num_hidden_layers, self.config.num_attention_heads))
 
+        if indices_prng_key is None:
+            indices_prng_key = jax.random.PRNGKey(0)
```

Comment:
We shouldn't default the PRNG key to 0. Let's use the same logic that we employ for `dropout` and create an RNG that we assign to `indices`.